README.md

```diff
@@ -79,69 +74,8 @@ I'm open to adding more drivers, feel free to file an issue.
 ## To use from Kubernetes:
 
 ### Using Deployment and Service
-Follow this [instruction](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/Kubernetes.md)
+Follow this [instruction](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/master/cloudsql).
```

Comment:
can you:
1. distinguish sidecar method vs Service method in the titles.
2. add a note here about the fact that we don’t advise using Service for the proxy and we recommend the sidecar method.